pkg/client/proxy.go

```diff
@@ -98,6 +98,11 @@ const (
 	GRPCServiceTimeout = 3 * time.Minute
```

Comment:
Should we have the timeout longer than [the one used in the proxy server](https://github.com/longhorn/longhorn-engine/blob/a63f0e0514872221c05ce6e2153a9164fa426f24/pkg/controller/client/controller_client.go#L40) to avoid the backend operation timeout earlier than this. ref to [link](https://github.com/longhorn/longhorn-manager/pull/1348#discussion_r884454508)

Reply:
Thank you. Updated.